### DISCUSSION TITLE
어제 미팅 때 잠시 말씀 드렸던 펄어비스의 개발 사례입니다.

### DISCUSSION THREAD

**Discussion by Joseph-Cha** [ThinkAboutSoftware/AcademicConference#234] [General]
![image](https://user-images.githubusercontent.com/75019048/174352503-aab462dd-8393-4018-a66f-a95e5e143bc3.jpeg)

[기사 전문](https://m.inven.co.kr/webzine/wznews.php?idx=208745)

기능 단위로 프론트와 서버 작업을 동시에 하는 건 정말 도입해볼만한 것 같습니다.

**Comment by jongfeel** [comment]
오 실제로 하는 곳이 있군요.

한번 해보고 싶다는 생각이...
